src/WebJobs.Script.WebHost/Controllers/ExtensionsController.cs

```diff
@@ -57,10 +57,10 @@ public async Task<IActionResult> Get()
 
         [HttpPost]
         [Route("admin/host/extensions")]
-        public Task<IActionResult> Post([FromBody]ExtensionPackageReferenceWithActions package) => InstallExtension(package);
+        public Task<IActionResult> Post([FromBody] ExtensionPackageReferenceWithActions package) => InstallExtension(package);
 
-        [HttpPut("{id}")]
-        [Route("admin/host/extensions")]
+        [HttpPut]
+        [Route("admin/host/extensions/{id}")]
```

Comment:
@soninaren I vaguely remember portal using either this or the `POST` API above to install extensions, right? Do you know if this change will affect that in any way ?

Reply:
Portal URI for extensions is [here](https://github.com/Azure/azure-functions-ux/blob/a0eec3ef46e5185a3c3539a327b83993354d4d59/client/src/app/shared/services/function-app.service.ts#L975). It's using the POST route which doesn't take an ID. However, if you query prod, you'll see some calls to the PUT route with an ID. Not clear if those are just testing.

However, we need to be sure that even though we're not using this ID for PUT, we have to make sure that we don't break the contract. Does the current route only match if an ID is specified? Or does it match both with/without? Need to test before changes and verify we don't change this.

Reply:
Confirmed that the above API can be called by either 

`PUT http://<appName>.azurewebsites.net/123`
or 
`PUT http://<appName>.azurewebsites.net/admin/host/extensions`

From logs, I don't see anything that's using the second API except the ones I've been testing. I just left the second API there and updated tests accordingly.